.github/workflows/allthekeeps-testnet.yml

```diff
@@ -0,0 +1,103 @@
+name: AllTheKeeps / Testnet
+
+on:
+  push:
+    branches:
+     - allthekeeps-on-ropsten
+  pull_request:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: 'Environment for workflow execution'
+        required: false
+        default: 'dev'
+      upstream_builds:
+        description: 'Upstream builds'
+        required: false
+      upstream_ref:
+        description: 'Git reference to checkout (e.g. branch name)'
+        required: false
+        default: 'main'
+
+jobs:
+  dapp-build-and-publish:
+    if: github.event_name != 'pull_request'
```

Comment:
In the case of `pull_request` we should build but not publish, it looks like we don't execute the workflow at all now for `pull_request`.

Reply:
We do the same thing for `tbtc dapp` & `keep dashboard`. You think running the flows "on push" won't be enough?

Reply:
Kuba's right here. For `tbtc dapp` & `keep dashboard` we have
```
    if: |
      github.event_name != 'pull_request'
        || needs.dapp-detect-changes.outputs.path-filter == 'true'
```
and the latter part of this condition is responsible for execution of job for `pull_request` (`path-filter` is set to `true` for workflows triggered by PRs affecting filtered paths).
In the `allthekeeps` workflow we want to execute the job for all pull requests, no matter which files are modified in them.

Reply:
ok, we can run on pull request as well. https://github.com/keep-network/allthekeeps/pull/1/commits/a350667bf11ed6b1aafd499edf852528d93ba5ea